R/import_data.R

```diff
@@ -154,7 +154,7 @@ changeToDatatype.iNZclass.factor <- function(obj){
   
   user.data.frame[[column.number]] <- as.factor(user.data.frame[[column.number]])
   
-  if ("metadata" %in% attributes(user.data.frame)){
+  if ("metadata" %in% names(attributes(user.data.frame))){
```

Comment:
Nothing wrong with this, but `%in%` can be a little slow (it checks `"metadata"` against every element of `names(...)`. A faster way is just to check if the requested attribute exists: `! is.null( attr(user.data.frame, "metadata") )`.